examples/mariadb-claim.yaml

```diff
@@ -4,6 +4,8 @@ metadata:
   name: platform-ref-aws-db-mariadb
 spec:
   parameters:
+    deletionPolicy: Delete
```

Comment:
Could you explain why we need explicit `deletionPolicy` to be exposed in customer abstraction?

Reply:
@stevendborrelli asked for it to be included. Going forward, I think we should keep PRs to the original intent and create new ones for new requests.